framework/src/utils/MooseUtils.C

```diff
@@ -553,4 +553,11 @@ wildCardMatch(std::string name, std::string search_string)
     return false;
 }
 
+std::string
+toUpper(std::string name)
+{
+  std::transform(name.begin(), name.end(), name.begin(), ::toupper);
+  return name;
```

Comment:
OK - I see... Hmm, So the input and output of this function end up being the same. If you are going to modify the input anyway, you could just make this a void function and have the passed in value modified directly (non-const reference). Alternatively you could take a const reference and return the modified string. I'm just not sure that I like the weird combination of modifying both the input and returning a copy of the modified input as the output too! One of these copies should be eliminated.